packages/flutter_localizations/test/material/translations_test.dart

```diff
@@ -535,9 +535,18 @@ void main() {
   // Regression test for https://github.com/flutter/flutter/issues/110451.
   testWidgets('Finnish translation for tab label', (WidgetTester tester) async {
     const Locale locale = Locale('fi');
-    expect(GlobalCupertinoLocalizations.delegate.isSupported(locale), isTrue);
+    expect(GlobalMaterialLocalizations.delegate.isSupported(locale), isTrue);
```

Comment:
why this change?

Reply:
Because this was a test I wrote some months ago (https://github.com/flutter/flutter/pull/130333/files) and where I made a cut and paste mistake (in test/material/translations_test.dart, we should rely on `GlobalMaterialLocalizations`, in that PR I updated both the cupertino test and the material test, and I wrongly copy/pasted this line.